streams/src/main/java/org/apache/kafka/streams/state/internals/ThreadCache.java

```diff
@@ -71,6 +72,22 @@ public long flushes() {
         return numFlushes;
     }
 
+    public void resize(final long newCacheSizeBytes) {
+        final boolean shrink = newCacheSizeBytes < this.maxCacheSizeBytes;
```

Comment:
nit: we can remove `this.` now (same next line)

Reply:
yep